templates/telemetry_derived/clients_daily_v6/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+friendly_name: Clients Daily
+description: >
+  A daily aggregate of each Firefox client,
+  partitioned by day
+owners:
+  - relud@mozilla.com
+labels:
+  application: firefox
+  refresh: daily
+  version: 6
+  incremental: true
```

Comment:
BQ labels don't natually suit boolean values; IIUC this will be a string value "true". Maybe `schedule: daily` would be more extensible than `incremental: true`. I need to read the proposal again to see how much the particular labels were discussed.

Reply:
True. As far as I can tell, the specific labels aren't really discussed in the proposal. I just copied the example for `clients_daily` from there for testing.